core/src/main/java/nl/nn/adapterframework/jta/btm/BtmDataSourceFactory.java

```diff
@@ -23,16 +23,26 @@
 import org.springframework.jdbc.datasource.DelegatingDataSource;
 
 import bitronix.tm.resource.jdbc.PoolingDataSource;
+import lombok.Getter;
+import lombok.Setter;
 import nl.nn.adapterframework.jndi.JndiDataSourceFactory;
 
 public class BtmDataSourceFactory extends JndiDataSourceFactory implements DisposableBean {
 
+	private @Getter @Setter int minPoolSize=0;
+	private @Getter @Setter int maxPoolSize=20;
+	private @Getter @Setter int maxIdleTime=60;
+	private @Getter @Setter int maxLifeTime=0;
```

Comment:
Is het niet mooier hier `AppConstants.getInstance().getProperty(...);` van te maken, dan blijft de spring configuration xml kleiner, maar is het toch instelbaar?